proto/axelar/nexus/v1beta1/events.proto

```diff
@@ -43,3 +43,8 @@ message RateLimitUpdated {
   google.protobuf.Duration window = 3
       [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
 }
+
+message NewGeneralMessage {
```

Comment:
Since `GeneralMessage` is a type, I don't like using `NewGeneralMessage` for the event name as it can collide with the constructor for that type in the future. Perhaps `MessageSubmitted` (we've also avoided using GeneralMessage for the naming in most places)?